.github/workflows/release.yml

```diff
@@ -326,3 +326,38 @@ jobs:
           target-folder: ${{ env.DOCS_VERSION_PATH }} # The folder the action should deploy to.
           commit-message: publish documentation
           clean: false # Releasing a new version is about creating a new directory, so we don't want to clean up the root.
+
+  delete-nightly-release:
+    runs-on: ubuntu-latest
+    needs: [deploy]
+
+    steps:
+      - name: Delete nightly release
+        uses: actions/github-script@v5
+        with:
+          github-token: ${{ secrets.gh_access_token }}
+          script: |
+            const releases = await github.rest.repos.listReleases({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+
+            const nightlyRelease = releases.data.find(r => r.tag_name.startsWith('nightly'))
```

Comment:
why startsWith and not equality?

Reply:
fixed

Reply:
I still see it in your current branche (83b028a).  BTW commit d138345d does not do what it claims.

Reply:
This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository.

Reply:
Same for the second one

Reply:
They are on your fork, and displayed when you click on the links; I wall update them anyway

Reply:
still not fixed